app/home.tmpl

```diff
@@ -146,17 +150,62 @@
                     {{ $path = regexReplaceAll ".*?Path(Prefix)?\\(\\`([a-z0-9-\\/]+)\\`\\).*" $host_rule "$2" }}
                 {{end}}
 
-                {{ $entrypoint_label := print "traefik.http.routers." $service_name ".entrypoints" }}
-                {{ $entrypoint := index $container.Labels $entrypoint_label }}
-                {{ $protocol := "" }}
-                {{ if eq $entrypoint "websecure" }}
-                    {{ $protocol = "https" }}
-                {{ else if eq $entrypoint "web" }}
-                    {{ $protocol = "http" }}
-                {{ else }}
-                    {{/* entrypoint not supported */}}
+            {{ $entrypoint_label := print "traefik.http.routers." $service_name ".entrypoints" }}
+            {{ $entrypoints := index $container.Labels $entrypoint_label }}
+            {{ $protocol := "" }}
+
+            {{ $entrypoint_label := print "traefik.http.routers." $service_name ".entrypoints" }}
+            {{ $entrypoints := index $container.Labels $entrypoint_label }}
+            {{ $protocol := "" }}
+
+            {{/* Check if entrypoints label is defined and split into an array */}}
+            {{ if $entrypoints }}
+                {{ $entrypoints_list := split $entrypoints "," }}
+
+                {{/* Flag to determine if a matching entrypoint was found */}}
+                {{ $entrypoint_found := false }}
+
+                {{/* Check if entrypoint is in the HTTPS entrypoints list */}}
+                {{ range $entrypoints_list }}
+                    {{ $current_entrypoint := . }}
+                    {{ $is_https := false }}
+                    {{ $is_http := false }}
+
+                    {{ range $https_entrypoints }}
+                        {{ if eq $current_entrypoint . }}
+                            {{ $is_https = true }}
+                            {{break}}
+                        {{ end }}
+                    {{ end }}
+
+                    {{ if $is_https }}
+                        {{ $protocol = "https" }}
+                        {{ $entrypoint_found = true }}
+                        {{break}}
+                    {{ else }}
+                        {{ range $http_entrypoints }}
+                            {{ if eq $current_entrypoint . }}
+                                {{ $is_http = true }}
+                                {{break}}
+                            {{ end }}
+                        {{ end }}
+
+                        {{ if $is_http }}
+                            {{ $protocol = "http" }}
+                            {{ $entrypoint_found = true }}
+                            {{break}}
+                        {{ end }}
+                    {{ end }}
```

Comment:
this else block can be removed and the code moved after the if.
if the condition is true, then the `break` statement at line 184 will make the execution return

it should be something like
```
{{ if $is_https }}
  {{ $protocol = "https" }}
  {{ $entrypoint_found = true }}
  {{break}}
{{ end }}
{{ range $http_entrypoints }}
...
```